src/include/miopen/conv/problem_description.hpp

```diff
@@ -281,6 +283,7 @@ struct ProblemDescription
     }
 
     static std::string table_name() { return "config"; }
+#if 0
```

Comment:
This should be elaborated before merging, right?

Reply:
Yea, why are we adding in disabled code?

Reply:
Please see issue #266 . I have undid this change, and we can discuss the problem over there.